### PR TITLE
Managing error in represenation of time

### PR DESCRIPTION
**Pull request by heysadboy**
Earlier it used to show 00:00 pm after 11:00 am. Showing time like 12:00 pm now. Solving this https://github.com/fossasia/open-event-android/issues/1448. See the screenshot to know more,

screenshot_2017-05-10-05-28-27

### CODE REVIEW

**Comment by iamareebjamal** [comment]
Isn't there some other way than string manipulation to achieve that? 

Looks hacky

**Comment by heysadboy** [comment]
@iamareebjamal If we do it in ISO conversions or anywhere else it would change the sorting order. This is the only anomaly I found so I though it would be better to just do it at the time of display.

File: android/app/src/main/java/org/fossasia/openevent/adapters/DayScheduleAdapter.java [hunk 1]
```diff
@@ -96,6 +96,15 @@ public void onBindViewHolder(DayScheduleViewHolder holder, int position) {
         String startTime = ISO8601Date.get12HourTime(ISO8601Date.getDateObject(currentSession.getStartTime()));
         String endTime = ISO8601Date.get12HourTime(ISO8601Date.getDateObject(currentSession.getEndTime()));
 
+        /*We are following 12 hour format so changing 00 PM to 12 PM */
```
Comment (Shailesh351) [inline android/app/src/main/java/org/fossasia/openevent/adapters/DayScheduleAdapter.java]: No need to manipulate string manually. You can just change 'KK' to 'hh'  in get12HourTime function in ISO8601Date.java
KK  is for 0-11 am/pm
hh is for 1-12 am/pm

Reference : https://developer.android.com/reference/java/text/SimpleDateFormat.html

**Comment by heysadboy** [comment]
@Shailesh351 @iamareebjamal Thanks for reviewing, please see.

**Pull request merged**